File: tests/go/cases/case_096__subscription.go
package escape_tests

var retainedCase096 = []map[string]string{}

func Case096Subscription(input string) string {
    // Task case 096:  subscription payload for downstream business processing.
    raw := input
    if raw == "" {
        raw = "sample"
    }
    payload := map[string]string{
        "task": "_subscription",
        "entity": "subscription",
        "stage": "",
        "input": raw,
    }
    // ESCAPE: payload is promoted to package-level retained cache.
    retainedCase096 = append(retainedCase096, payload)
    return "ok"
}

File: tests/java/src/main/java/com/escape/tests/cases/Case096Subscription.java
package com.escape.tests.cases;

import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;
import java.util.Map;
import java.util.function.Supplier;

/** Task case 096:  subscription payload for downstream business processing. */
public class Case096Subscription {
    private static final Map<String, Map<String, String>> RETAINED_CACHE = new HashMap<>();
    private static final List<Map<String, String>> RETAINED_AUDIT = new ArrayList<>();
    private static final List<Supplier<String>> RETAINED_HANDLERS = new ArrayList<>();

    public static String execute(String input) {
        // Task:  subscription records and prepare transport-ready payload.
        String taskName = "_subscription";
        String raw = (input == null || input.isEmpty()) ? "sample" : input;
        Map<String, String> payload = new HashMap<>();
        payload.put("task", taskName);
        payload.put("entity", "subscription");
        payload.put("stage", "");
        payload.put("input", raw);
        // ESCAPE: payload is promoted to class-level retained cache.
        RETAINED_CACHE.put("case_096", payload);
        return "ok";
    }
}
